experiments/compositions/composition/internal/controller/expander_reconciler.go

```diff
@@ -221,6 +222,17 @@ func (r *ExpanderReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 
 	// ------------------- APPLIER SECTION -----------------------
 	stagesApplied := []string{}
+	expanderLogsEnabled := false
+	var err error
+	value, exist := inputcr.GetAnnotations()["composition-expander-stage-logs"]
+	if !exist {
+		logger.Info("annotation does not exist.")
```

Comment:
log what annotation doesnt exist.

Reply:
I would log if the annotation exists and not log if it doesnt exist (more common use case)

Reply:
Make sense.